doc/API.md

```diff
@@ -51,18 +51,29 @@ The Datadog tracing module defines the following configuration directives.
 ### `datadog_service_name`
 - **syntax** `datadog_service_name <name>`
 - **default**: `nginx`
-- **context**: `http`
+- **context**: `http`, `server`
 
 Set the service name to associate with each span produced by this module.
+May overwrite `DD_SERV` environment variable.
```

Comment:
Should this be `DD_SERVICE`?

```suggestion
May overwrite `DD_SERVICE` environment variable.
```

Reply:
addressed in b558dc79273d05d6ab4a26f008a79a1c7d78c812